tests/shared/pages/InboxPage.ts

```diff
@@ -12,7 +12,14 @@ export class ModalPage {
   }
 
   async load() {
-    await this.page.goto(this.baseURL)
+    await this.page.goto(`${this.baseURL}`)
```

Comment:
No need template literals I think:
```suggestion
    await this.page.goto(this.baseURL)
```